LambentLight/Landing.cs

```diff
@@ -422,6 +422,31 @@ private void RefreshInstalledResources()
             UninstallerListBox.Fill(((DataFolder)DataBox.SelectedItem).InstalledResources);
         }
 
+
         #endregion
+
+        private async void RestartServer_Click(object sender, EventArgs e)
+        {
+            ProcessManager.Stop();
+            Locked = false;
+
+            // Ensure that we have an item available
+            if (BuildsListBox.SelectedItem == null)
+            {
+                // If not, notify the user and return
+                Logger.Info("You have not selected a FiveM/CitizenFX server build");
+                return;
+            }
+            // Do the same with server data folders
+            if (DataBox.SelectedItem == null)
+            {
+                // Notify and return
+                Logger.Info("You have not selected a Server Data folder");
+                return;
+            }
+
```

Comment:
I don't think that this is required because we already did a check prior to starting the server.

Reply:
Will reevaluate and resubmit, thank you